product_docs/docs/pgd/5/quick_start_docker.mdx

```diff
@@ -157,27 +155,27 @@ tpaexec configure democluster \
   --hostnames-unsorted
 ```
 
-We included options to specify using Docker, a single location, and three data nodes running [EDB Postgres Advanced Server](/epas/latest/). By default, PGD also configures two [PGD Proxy](routing/proxy/) nodes and a [Barman](backup#physical-backup) node for backup. Rocky Linux is the default image for all nodes.
+These options specify to use Docker, a single location, and three data nodes running [EDB Postgres Advanced Server](/epas/latest/). By default, PGD also configures two [PGD proxy](routing/proxy/) nodes and a [Barman](backup#physical-backup) node for backup. Rocky Linux is the default image for all nodes.
```

Comment:
Also like the AWS version:

These options specify that TPA should create a configuration that uses Docker, in a single location, with three data nodes...